nostr-java-api/src/main/java/nostr/api/factory/impl/NIP15Impl.java

```diff
@@ -4,23 +4,19 @@
  */
 package nostr.api.factory.impl;
 
-import java.util.ArrayList;
-import java.util.List;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NonNull;
 import nostr.api.factory.EventFactory;
 import nostr.event.BaseTag;
-import nostr.event.impl.CreateOrUpdateProductEvent;
-import nostr.event.impl.CreateOrUpdateStallEvent;
-import nostr.event.impl.CustomerOrderEvent;
-import nostr.event.impl.MerchantRequestPaymentEvent;
-import nostr.event.impl.NostrMarketplaceEvent;
-import nostr.event.impl.VerifyPaymentOrShippedEvent;
+import nostr.event.impl.*;
```

Comment:
Better to import just the specifics dependencies in use than all them ("*").

Reply:
IntelliJ does this automatically. I'll make the necessary changes and resubmit.

Reply:
You can config this in your IDE